CommonMark/CommonMarkException.cs

```diff
@@ -8,7 +8,7 @@ namespace CommonMark
     /// <summary>
     /// An exception that is caught during CommonMark parsing or formatting.
     /// </summary>
-#if v2_0 || v3_5 || v4_0 || v4_5
+#if v2_0 || v3_5 || v4_0 || (v4_5 && !portable)
```

Comment:
I don't like that here the portable 4.5 will differ in functionality from the separate 4.5 - they should be the same